packages/build/src/steps/get.js

```diff
@@ -7,18 +7,37 @@ import { bundleFunctions } from '../plugins_core/functions/index.js'
 // Get all build steps
 export const getSteps = function (steps) {
   const stepsA = addCoreSteps(steps)
-  const stepsB = sortSteps(stepsA)
+  const stepsB = sortSteps(stepsA, EVENTS)
   const events = getEvents(stepsB)
   return { steps: stepsB, events }
 }
 
+// Get all dev steps
+export const getDevSteps = function (command, steps) {
+  const devCommandStep = {
```

Comment:
This is an inline core step that just wraps the callback we receive in `startDev`.